modules/developing/pages/clients/flink.adoc

```diff
@@ -0,0 +1,476 @@
+= Apache Flink
+
+Apache Flink is an open source stream processing framework with powerful stream- and batch-processing capabilities.
```

Comment:
"...stream- and batch-processing..." Let's emphasize real-time nature of Flink. Maybe something like "... with powerful real-time, unbounded data processing capabilities (aka, streaming). For more information about Flink, please refer to the official Apache Flink [documentation](https://flink.apache.org/))